cohortextractor/update_vmp_mapping.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import requests
+
+from .mssql_utils import mssql_dbapi_connection_from_url
+
+
+def update_vmp_mapping():
+    """Update VmpMapping table with latest data from OpenCodelists"""
+
+    mapping = [tuple(record) for record in get_vmp_mapping()]
+    temporary_database = os.environ["TEMP_DATABASE_NAME"]
+    conn = mssql_dbapi_connection_from_url(os.environ["DATABASE_URL"])
+    cursor = conn.cursor()
+    cursor.execute(
+        f"""
+IF OBJECT_ID('{temporary_database}..VmpMapping', 'U') IS NOT NULL
+    DROP TABLE {temporary_database}..VmpMapping
+
+CREATE TABLE {temporary_database}..VmpMapping (
+    id VARCHAR(18) COLLATE Latin1_General_CI_AS,
+    prev_id VARCHAR(18) COLLATE Latin1_General_CI_AS,
+)
+        """
+    )
+    cursor.executemany(
+        f"INSERT INTO {temporary_database}..VmpMapping VALUES (%s, %s)",
+        mapping,
+    )
+    conn.commit()
+    conn.close()
+
+
+def get_vmp_mapping():
+    rsp = requests.get("https://opencodelists-proxy.opensafely.org/api/v1/dmd-mapping/")
```

Comment:
The `.json()` call is probably sufficient error checking here, but out of paranoia I generally like to call `rsp.raise_for_status()` after each request to make sure I'm not working with an error response.